<ui version="4.0" >
 <class>frmPartSpectController</class>
 <widget class="QWidget" name="frmPartSpectController" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="grbxFile" >
     <property name="title" >
      <string>File</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lblFileName" >
        <property name="text" >
         <string>File long name goes here </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="pbtnOpenFile" >
        <property name="text" >
         <string>Open</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>66</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="grbxParticles" >
     <property name="title" >
      <string>Show spectrum</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" colspan="2" >
       <widget class="QRadioButton" name="rbtnBaryon" >
        <property name="text" >
         <string>baryon</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="rbtnMeson" >
        <property name="text" >
         <string>meson</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="rbtnE" >
        <property name="text" >
         <string>e</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QRadioButton" name="rbtnNeutron" >
        <property name="text" >
         <string>neutron</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QRadioButton" name="rbtnGamma" >
        <property name="text" >
         <string>gamma</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QRadioButton" name="rbtnNucleus" >
        <property name="text" >
         <string>nucleus</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QRadioButton" name="rbtnLepton" >
        <property name="text" >
         <string>lepton</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QRadioButton" name="rbtnPi" >
        <property name="text" >
         <string>pi</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2" >
       <widget class="QRadioButton" name="rbtnProton" >
        <property name="text" >
         <string>proton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>68</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
